assignment08/js/clock.js

```diff
@@ -0,0 +1,87 @@
+window.addEventListener('load', () => {
+ class Clock   {   
+
+startClock ()  {
+    //YOUR CODE SHOULD START BELOW THIS LINE
+const clock = document.getElementById('clock');
+const context = clock.getContext('2d');
+let radius = clock.height / 2;
+context.translate(radius, radius);
+radius = radius * 0.90;
+drawClock();
```

Comment:
I would move this to the end of the function (after all the innerfunctions are defined) and have the clock draw at an interval so that it updates with each second:
setInterval(drawClock, 1000)